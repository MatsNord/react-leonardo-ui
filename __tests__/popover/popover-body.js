import React from 'react';
import renderer from 'react-test-renderer';

import Popover from '../../src/popover';

test('Popover body renders correctly', () => {
  const tree = renderer.create(
    <Popover.Body>Body content</Popover.Body>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Popover body passes props correctly', () => {
  const tree = renderer.create(
    <Popover.Body className="my-class" data-x="x" title="Body title">
      Body title
    </Popover.Body>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
