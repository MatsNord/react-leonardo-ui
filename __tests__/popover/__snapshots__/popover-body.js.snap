// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popover body passes props correctly 1`] = `
<div
  className="lui-popover__body my-class"
  data-x="x"
  title="Body title"
>
  Body title
</div>
`;

exports[`Popover body renders correctly 1`] = `
<div
  className="lui-popover__body"
>
  Body content
</div>
`;
